sr_robot_launch/scripts/sr_ur_release

```diff
@@ -0,0 +1,193 @@
+        return fault
+
+    def check_arms_robot_mode(self, arms):
+        arm_needs_starting = False
+        for arm in arms:
+            robot_mode = self.get_robot_mode(arm)
+            if robot_mode == RobotMode.IDLE or robot_mode == RobotMode.POWER_OFF:
+                rospy.loginfo("Starting arm: %s", arm)
+                arm_needs_starting = True
+        return arm_needs_starting
+
+    def clear_arms_popups(self, arms):
+        for arm in arms:
+            self.call_dashboard_service(arm, "close_safety_popup")
+        rospy.sleep(2)
+        for arm in arms:
+            self.call_dashboard_service(arm, "close_popup")
+        rospy.sleep(2)
+
+    def get_program_state(self, arm):
+        try:
+            play_mode_service = rospy.ServiceProxy("/" + arm + "_sr_ur_robot_hw/dashboard/program_state", GetProgramState)
+            play_msg = play_mode_service()
+            return play_msg
+        except rospy.ServiceException, e:
+            rospy.logerr("Service call to 'program_state' failed for arm %s. %s", arm, e)
+
+    def check_program_loaded_arms(self, arms):
+        sleep_time = False
+        for arm in arms:
+            if not rospy.get_param("/" + arm + "_sr_ur_robot_hw/headless_mode") :
+                play_msg = self.get_program_state(arm)
+                if play_msg.program_name == "null":
+                    rospy.loginfo("Not in headless mode. Loading program: %s for arm: %s", self.external_control_program_name, arm)
+                    self.load_external_control_program(arm)
+                    sleep_time = True
+        return sleep_time
+
+    def check_program_playing_arms(self, arms):
+        sleep_time = False
+        for arm in arms:
+            if not rospy.get_param("/" + arm + "_sr_ur_robot_hw/headless_mode") :
+                play_msg = self.get_program_state(arm)
+                if play_msg.state.state == ProgramState.STOPPED or play_msg.state.state == ProgramState.PAUSED:
+                    rospy.loginfo("Not in headless mode. Starting program: %s for arm: %s", play_msg.program_name, arm)
+                    self.call_dashboard_service(arm, "play")
+                    sleep_time = True
+            else:
+                rospy.loginfo("Headless mode detected, resending robot program to arm: %s", arm)
+                self.call_service(arm, "resend_robot_program")
+                rospy.sleep(2)
+                self.call_service(arm, "resend_robot_program")
+                sleep_time = True
+        return sleep_time
+
+    def release_arm(self):
+        try:
+            rospy.loginfo("Pedal release signal received. Checking arms.")
+            rospy.loginfo("Checking e-stops ...")
+            self.check_arms_e_stops(self.arms)
+            rospy.loginfo("Checking protective stops ...")
+            self.check_arms_protective_stop(self.arms)
+            rospy.loginfo("Checking for faults ...")
+            if self.check_arms_fault(self.arms):
```

Comment:
this name is misleading. Its not only checking fault but also resetting

Reply:
Is check_fix_... ok?